src/views/portal/Message.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "views/portal/Portal.module.scss";
+
+interface MessageProps {
+	status: string;
+}
```

Comment:
Note: we enumerated some of the applicant statuses while working on the admin portal as part of #72, but I'm also reorganizing more of it while working on #84. As of now, we'll be separating status from the decision, more to come in another pull request soon.